internal/grpctest/grpctest.go

```diff
@@ -53,13 +55,18 @@ func RunSubTests(t *testing.T, x interface{}) {
 	setup := getTestFunc(t, xv, "Setup")
 	teardown := getTestFunc(t, xv, "Teardown")
 
+	logger.setLevel(infoLog)
```

Comment:
The level and verbosity are set at initialization time.  I think you could delete these setters, because they don't need to be changed dynamically.

Reply:
Deleted the setters.